includes/polling/os/apc.inc.php

```diff
@@ -1,94 +1,20 @@
 <?php
 
-// PDU
-$serial = trim(snmp_get($device, 'PowerNet-MIB::rPDUIdentSerialNumber.0', '-OQv'), '"');
+$apc_serial = snmp_get_multi_oid($device, 'rPDUIdentSerialNumber.0 atsIdentSerialNumber.0 upsAdvIdentSerialNumber.0 sPDUIdentSerialNumber.0 airIRRCUnitIdentSerialNumber.0 isxModularPduIdentSerialNumber.0 airIRRP100UnitIdentSerialNumber.0 airIRRP500UnitIdentSerialNumber.0', '-OUQs', 'PowerNet-MIB');
```

Comment:
Yes I've taken this from your PR and it seems to work fine with my Air Conditionner check. I will test it with APC PDU, Automatic Transfer Switch and Netbotz devices.